### PR TITLE
Feature/updates

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-redirect-pr-14 in nhsuk-dev succeeded (http://gp-redirect-pr-14.dev.beta.nhschoices.net)

File: scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+[`pre-bootstrap`](pre-bootstrap)
+Directs towards base development machine setup.
+
+[`bootstrap`](bootstrap)
+Installs project's direct dependencies e.g. npm packages.
+
+[`start`](start)
+Starts the application a Docker container. Available at:
+`http://localhost:3000`
```
Comment (markysoft) [inline scripts/README.md:18]: The readme file says the default port is 3000, but there is no default set in the code. If PORT is not set you get the message 'app listening on port undefined'

**Review by markysoft — CHANGES_REQUESTED** [review]
There is no default port set if the PORT env var is missing.

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-redirect-pr-14 in nhsuk-dev succeeded (http://gp-redirect-pr-14.dev.beta.nhschoices.net)

File: scripts/README.md (hunk 1 above)
Comment (st3v3nhunt) [reply scripts/README.md:18]: Fixed

**Review by markysoft — APPROVED** [review]

**Pull request merged**